gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.6.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.0-all.zip
```

Comment:
I think we can go with 8.4

Reply:
I used the below command now to include all the script files:
`./gradlew wrapper --gradle-version=8.4`